### PR TITLE
feat(skymp5-front): add Storybook

### PR DESCRIPTION
**Pull request by JustCaptcha**
Closes: [399](https://github.com/skyrim-multiplayer/skymp/issues/399), [560](https://github.com/skyrim-multiplayer/skymp/issues/560).
Depends on [471](https://github.com/skyrim-multiplayer/skymp/issues/471)

Changes:
- From now on, we should use [yarn](https://github.com/yarnpkg/yarn) because npm could not resolve the dependency vulnerability issue in the trim package. Issue - [Vulnerability on @storybook/addon-docs related with trim](https://github.com/storybookjs/storybook/issues/14603)
- FrameButton was slightly rewritten.
- Resolved fonts issue.

image

### CODE REVIEW

File: skymp5-front/src/components/FrameButton/index.tsx [hunk 1]
```diff
@@ -1,55 +1,93 @@
...
+import FrameButtonStartImage from './img/frame_button_start.svg';
 import './styles.scss';
 
 interface IFrameButtonProps extends IDefaultUIComponentProps {
     variant: string;
 }
 
 interface IButtonItemProps extends IDefaultUIComponentProps {
-    name: string;
+    // name: string;
```
Comment (Pospelove) [inline skymp5-front/src/components/FrameButton/index.tsx:18]: commented code

File: skymp5-front/webpack.config.ts [hunk 2]
```diff
@@ -51,10 +51,10 @@ module.exports = {
         test: /\.(png|svg|jpg|gif)$/,
         use: "file-loader",
       },
-      {
-        test: /\.(woff|woff2|eot|ttf|otf)$/,
-        use: "file-loader",
-      },
+      // {
```
Comment (Pospelove) [inline skymp5-front/webpack.config.ts:54]: and here

**Comment by nic11** [comment]
You commited `.DS_Store`

**Review by Pospelove — CHANGES_REQUESTED** [review]
Please fix json in skymp/unit:
```

: D:\a\skymp\skymp\unit\DistContentsTest.cpp(66): FAILED:
2: due to unexpected exception with message:
2:   Unexpected contents of 'D:/a/skymp/skymp/build/dist', see diff
2:   Excess files:
2:    + "server\\data\\ui\\00e4a2200c3e5ae71d67.ttf"
2:    + "server\\data\\ui\\268a0df3f0e2c23d1ea6.ttf"
2:    + "server\\data\\ui\\6f802bf60c545304b09e.ttf"
2:    + "server\\data\\ui\\855bfaf67c57745958d3.ttf"
2:    + "server\\data\\ui\\d213051bb6f28a25fb26.ttf"
2:    + "server\\data\\ui\\f4d6dc3e7665a2c2fc6d.png"
2:    + "server\\data\\ui\\fdf086661e1600f95a34.ttf"
2:   Missing files:
2:    - "server/data/ui/00e4a2200c3e5ae71d67e16d7990c68f.ttf"
2:    - "server/data/ui/268a0df3f0e2c23d1ea6e0e024316b34.ttf"
2:    - "server/data/ui/6f802bf60c545304b09e5fed8484f5c7.ttf"
2:    - "server/data/ui/855bfaf67c57745958d3a09367a4ccb8.ttf"
2:    - "server/data/ui/d213051bb6f28a25fb26372818032cf4.ttf"
2:    - "server/data/ui/fdf086661e1600f95a34561c377e0a9e.ttf"
```

Also remove DS_Store file

**Review by Pospelove — CHANGES_REQUESTED** [review]
oops we need to resolve conflicts again

**Review by Pospelove — APPROVED** [review]

**Pull request merged**